index.d.ts

```diff
@@ -49,16 +49,16 @@ type ZoneLetter = 'X'
 
 declare class UTMLatLng {
 
-Constructor(type?: ElipsoidType)
```

Comment:
Changed "Constructor" to "constructor" because constructors do not work when defined with an uppercase letter in TypeScript.